pom.xml

```diff
@@ -807,9 +807,56 @@ under the License.</licenseText>
             </dependency>
           </dependencies>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.8.1</version>
+          <configuration>
+            <activeRecipes>
+              <!-- Issue: https://github.com/apache/maven/pull/2277 -->
+              <!-- Fix: https://github.com/openrewrite/rewrite-static-analysis/pull/558 -->
+              <!-- <recipe>org.openrewrite.staticanalysis.ReplaceCollectToListWithToList</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateFields</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.UnnecessaryThrows</recipe> -->
+              <!-- <recipe>org.openrewrite.java.ShortenFullyQualifiedTypeReferences</recipe> -->
+              <recipe>org.openrewrite.java.RemoveUnusedImports</recipe>
```

Comment:
That one is unneeded, it's covered by spotless already.

Reply:
ok, nice yes. Then remove.